tumblrWatch/com/5yp/tumblrWatch/tumblr_utils.py

```diff
@@ -0,0 +1,24 @@
+import pytumblr
```

Comment:
Do you need this anymore? If so make it named something specific as it has hard coded values.